### DISCUSSION TITLE
I'd like to know the version of subclipse for svn1.14

### DISCUSSION THREAD

**Discussion by ybnn** [subclipse/subclipse#191] [Q&A]
**### plz help me!**
It's urgent

**Comment by markphip** [comment]
It is covered in the main docs: https://github.com/subclipse/subclipse/wiki

Install the latest version and make sure you choose the 1.14 JavaHL connector. If you are not on Windows then visit the JavaHL wiki page as it is up to you to install the native libraries that are used for your OS:  https://github.com/subclipse/subclipse/wiki#javahl-installation
